packages/fluentui/react-northstar/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -16,6 +16,12 @@ export const popupContentStyles: ComponentSlotStylesPrepared<PopupContentStylesP
         padding: v.pointerMargin,
       }),
     }),
+
+    ...(p.autoSize && {
+      // when autoSize is true, root will have inline style maxWidth and maxHeight set by popper
+      display: 'flex',
+      overflow: 'hidden',
```

Comment:
Still don't understand why this is necessary sorry

Reply:
The purpose is to make popup `content` no bigger than `root`. 

Initially I thought a `max-height` on `root` is good enough. But I found out the child(`content`) still overflows. Then I read about [this](https://stackoverflow.com/a/14263416/12667402). Shockingly when parent element has only `max-height` but not `height`, child's `max-height` computes to none. 
So I found another way to achieve the same goal, which is display flex.